news/_posts/translation/2021-07-01-do-you-speak-prestashop-june-2021.md

```diff
@@ -0,0 +1,121 @@
+| 2. | SNor42 | Hungarian | 5,671
+| 3. | SeongHyeon Cho (jaymz9634) | Korean | 3,067
+| 4. | Lauri Lepik (laurilllepik) | Estonian | 2,903
+| 5. | Cha (cafetango) | Chinese Traditional | 2,852
+| 6. | Stamatis (breezer) | Greek | 2,810
+| 7. | Patrik Lindblom (extendus) | Swedish | 2,792
+| 8. | Daniel Hlavacek (Hlavtox) | Czech | 2,666
+| 9. | David Gonzalez (davidglezz) | Spanish, | 2,310
+| 10. | Enzo Ferrara (ferribbotte) | Italian | 2,012
+ 
+#### Top 10 proofreaders in June:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | Zoran Tejic (zotamal) | Serbian (Latin) | 44,080
+| 2. | SNor42 | Hungarian | 6,325
+| 3. | Lauri Lepik (laurilllepik) | Estonian | 5,703
+| 4. | SeongHyeon Cho (jaymz9634) | Korean | 3,224
+| 5. | Cha (cafetango) | Chinese Traditional | 2,900
+| 6. | Enzo Ferrara (ferribbotte) | Italian | 2,827
+| 7. | Stamatis (breezer) | Greek | 2,810
+| 8. | Patrik Lindblom (extendus) | Swedish | 2,807
+| 9. | David Gonzalez (davidglezz) | Spanish | 2,666
+| 10. | Rodrigo Laurindo (rblaurin) | Portuguese, Brazilian | 2,545
+
+Congrats, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of June, just a few days after the new strings were added to Crowdin, PrestaShop 1.7.8 was already fully available (= 100% translated and validated) in **7 languages**. That’s great! Thank you for your involvement. :tada:
+ 
+| Chinese traditional | English | Estonian | French | 
+| Greek | Portuguese, Brazil | Swedish |
+
+Thank you all for your contribution! 
+
+## Languages with the best evolution
+
+The following languages had the best progress thanks to the translation community:
+ 
+* **Serbian (Latin)** (+36% to reach 97% translated and validated) 
+* **Czech** (+5% to reach 99% translated)
+* **Greek** (+5% to reach 100% translated and validated)
+
+Congratulations! And a special thanks to [Zoran Tejic](https://crowdin.com/profile/zotamal) who worked a lot on the Serbian project lately. Only 3% left before the language is fully available, almost there! Feel free to [contact me](https://crowdin.com/profile/julievrz) if you want to help the Serbian translators reach 100%.
+ 
+![Best translation progress in June 2021](/assets/images/2021/07/build-crowdin-progress-june21.png)
+
+A huge thank you to all the contributors!
+ 
+Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting could be going on, but the percentage of translation would not be modified (since it is working on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
```

Comment:
```suggestion
Of course, this is highlighting the languages that made some progress with new translations; but it does not mean that the languages that are not mentioned here are not active. Indeed, some editing and rewriting could be going on, but the percentage of translation would not be modified (since the work happens on strings that are already translated). So let's not forget about the work of other contributors! Thanks to you too!
```

Maybe better, I don't know